pkg/runtime/runtime.go

```diff
@@ -1285,12 +1286,16 @@ func (a *DaprRuntime) blockUntilAppIsReady(ctx context.Context) error {
 			break
 		}
 
+		counter++
 		select {
 		// Return
 		case <-ctx.Done():
 			return ctx.Err()
 		// prevents overwhelming the OS with open connections
-		case <-a.clock.After(time.Millisecond * 100):
+		case <-a.clock.After(time.Millisecond * 100): // .1 sec
```

Comment:
Should be 1000?

Reply:
he probably means the next line to print the logs slower than the sleep